java/client/build.gradle

```diff
@@ -234,3 +235,22 @@ jar {
     // placeholder will be renamed by platform+arch on the release workflow java-cd.yml
     archiveClassifier = "placeholder"
 }
+
+spotbugsMain {
+    reports {
+        html {
+            required = true
+            outputLocation = file("$buildDir/reports/spotbugs/main/spotbugs.html")
```

Comment:
Does it run on `./gradlew build`?

Reply:
Yes - works with all three - 
 - `./gradlew build`
 - `./gradlew :client:buildAll`
 - `./gradlew :client:buildAllRelease`